src/main/java/webserver/http/Cookie.java

```diff
@@ -0,0 +1,58 @@
+package webserver.http;
+
+import java.util.Arrays;
+import java.util.HashMap;
+import java.util.Map;
+import java.util.stream.Collectors;
+
+public class Cookie {
+
+    private final Map<String, String> cookies = new HashMap<>();
+
+    public Cookie() {
+
+    }
+
+    public Cookie(String cookieString) {
+        if (cookieString == null || cookieString.isEmpty()) return;
```

Comment:
저는 프레임워크에서 제공하는 util 기능으로 null 과 empty 체크를 하는 편입니다 ㅎㅎ
util 메소드만 호출하면 돼서 은근 사용하기 편하더라고요~
개인적인 취향이기 때문에 참고하시라고 코멘트 드립니다~
```suggestion
        if (StringUtils.isEmpty(cookieString)) return;
```
![image](https://user-images.githubusercontent.com/41741539/218357575-05fc7e04-c29f-4ec3-9557-6f890bd469d7.png)

Reply:
오... 이런 방법이 있었군요! 앞으로 자주 사용해보겠습니다!